brain/task_executor.py

```
@@ -419,6 +427,312 @@ def _extract_latest_user_intent(self, conversation: str) -> str:
+            (r"(?i)(password|passwd|pwd)\s*[:=]\s*\S+", r"\1=[REDACTED_PASSWORD]"),
+            (r"(?i)(password|passwd|pwd|密码|口令)\s*(?:is|是|=|:)\s*\S+", r"\1=[REDACTED_PASSWORD]"),
+            (r"(?i)(token|api[_-]?key|access[_-]?token|refresh[_-]?token)\s*[:=]\s*\S+", r"\1=[REDACTED_TOKEN]"),
+            (
+                r"(?i)(token|api(?:[\s_-]?key)|access(?:[\s_-]?token)|refresh(?:[\s_-]?token)|令牌|密钥|秘钥)\s*(?:is|是|=|:)\s*\S+",
+                r"\1=[REDACTED_TOKEN]",
+            ),
+            (r"(?i)\bsk-[a-z0-9_-]{10,}\b", "[REDACTED_TOKEN]"),
+            (r"(?i)(cookie)\s*[:=]\s*\S+", r"\1=[REDACTED_COOKIE]"),
+            (r"(?i)(cookie)\s*(?:[:=]|is|是)\s*\S+", r"\1=[REDACTED_COOKIE]"),
+            (
+                r"(?i)(\b(?:otp|pin|verification(?:\s+code)?|sms\s*code|one[-\s]?time(?:\s+password|\s+code)?|验证码|校验码|短信码|动态码)\b(?:\s*(?:is|为|是))?[\s:：=#-]{0,6})\d{4,8}\b",
```

Ruff (0.15.9)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)